jukebox/train.py

```diff
@@ -31,9 +33,13 @@ def log_aud(logger, tag, x, hps):
 
 def log_labels(logger, labeller, tag, y, hps):
     y = y.cpu().numpy()
-    txt = f''
+    attrs = ["artist", "genre", "lyrics"]
+    labels = []
     for item in range(y.shape[0]):
-        txt += labeller.describe_label(y)
+        meta = labeller.describe_label(y[item])
+        meta = [("sample", item), *list(map(lambda attr: (attr, meta[attr]), attrs))]
+        labels.append(json.dumps(OrderedDict(meta)))
+    txt = "\n".join(labels)
```

Comment:
fix merged in previous pull request, so you might need to resolve merge conflict here